### PR TITLE
added badges with links for npm version, downloads and licence

### PR DESCRIPTION
**Pull request by l92s**
issue #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,31 +26,29 @@ $ npm install @gh-conf/gh-conf-validate
 ## Usage
 
 ```javascript
-const { validatePath } = require('@gh-conf/gh-conf-validate');
+const { validatePath } = require("@gh-conf/gh-conf-validate");
```
Comment (arshadkazmi42) [inline README.md]: Have you change these intentionally? Or your edited did it?
We use single codes for strings

File: README.md [hunk 2]
```diff
@@ -23,31 +26,29 @@ $ npm install @gh-conf/gh-conf-validate
...
+const isValid = validatePath("./gh-conf-validate");
 console.log(isValid);
 // Output
 // true
 
-
 // Invalid path
 try {
-  validatePath()
+	validatePath();
```
Comment (arshadkazmi42) [inline README.md]: We use 2 spaces not tag spaces. Semicolon is fine though.

Same for below lines also

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
Looks great.
Just a couple of feedbacks, its good to go once those fixes are in place

File: README.md (hunk 1 above)
Comment (l92s) [reply README.md]: Sorry yes my editor auto-formats these, will correct asap :+1:

File: README.md (hunk 2 above)
Comment (l92s) [reply README.md]: Again my editor auto-formatting, will look asap!

File: README.md [hunk 3]
```diff
@@ -31,23 +34,21 @@ console.log(isValid);
...
 
   // Output
-  /**
-   * {
-   *   status: 'err',
-   *   message: 'No path input',
-   *   data: {}
-   * }
-   **/
+	/**
```
Comment (arshadkazmi42) [inline README.md]: lets make these also 2 spaces

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
Thanks for implementing the feedback, Can you make changes for these also. We will get this merged.

File: README.md (hunk 3 above)
Comment (l92s) [reply README.md]: this should be working with two spaces now, apologies as I don't know what my editor was trying to do with this spacing!

**Review by arshadkazmi42 — APPROVED** [review]
Looks good. Thanks 💯

**Pull request merged**